contracts/src/TeleporterTokenDestination.sol

```diff
@@ -289,7 +293,7 @@ abstract contract TeleporterTokenDestination is
      * to another destination bridge instance.
      */
     function _sendAndCall(
-        SendAndCallInput memory input,
+        SendAndCallInput calldata input,
```

Comment:
What's the reason for the change from `memory` here?

Reply:
Basically because we can. It's more gas efficient. We can do it in the destination because we don't have any calls to `_sendAndCall` that don't originate from the external `sendAndCall` call. We can't do it in the source, because the multihop `receiveTeleporterMessage` case can call `_sendAndCall`, so we don't have an immumtable calldata to pass there.